### ISSUE TITLE
[CJ4 0.7.1] Inconsistent VS mode functionality

### ISSUE THREAD

**Issue by linmar22** [Working-Title-MSFS-Mods/fspackages#542]
Most of the time VS mode works fine, but occasionally after using it, it seems to not disengage, or reset its reference vertical speed.

### Issue 1
Every once in a while, if you change the reference vertical speed while flying at the captured altitude, the ALT mode will, in essence, start fighting with the VS mode, like VS mode didn't disengage when it reached the desired altitude.

**Scenario:**
1. Set VS to -2500 and descend to desired alt.
2. Once the aircraft approaches the desired altitude, the VS mode disengages, ALT mode engages, and the aircraft begins to levels out.
3. Adjust the reference VS without turning VS mode on.

_**What should happen:**_
Reference VS changes, but the aircraft does not change attitude.

_**What actually happens:**_
Changing the reference VS causes the aircraft to oscillate, almost like its fighting with ALT mode.

### Issue 2
In addition to the above, sometimes VS mode will keep the reference vertical speed from the previous descent, so when you engage it, it violently dives into the desired VS. 

**In other words the scenario looks like so:**
1. Set VS to -2500 and descend to desired altitude.
2. Once you reach the desired altitude, the aircraft levels out and disengages the VS AP mode.
3.  Toggle VS mode ON again.

**_What should happen:_**
VS mode turns on with reference VS set to 0 FPM.

**_What actually happens:_**
VS mode turns on with -2500 FPM.

### Untested theory
If the reference vertical speed is changed after it begins leveling out, but before the aircraft captures the desired altitude, the reference VS is not reset to 0 in the back-end.

Feel free to contact me via Martus#4304 on your discord for questions or help diagnosing the issue.

**Comment by dga711** [comment]
Thanks for the feedback.
To me this sounds like default AP behavior that you would see in any plane.
As of now we have no control over these things.

We could maybe work on resetting the reference VS to workaround these "bugs" in the sim. We'll have to see.

**Comment by linmar22** [comment]
Weird I haven't noticed it in the other aircraft. In theory, resetting reference VS could even be done via an external application, listening/talking to simconnect. Hopefully asobo get their sh*t together on this one at some point. 👍🏻

**Comment by dga711** [comment]
Ok maybe not on any plane ^^ I just mean that we didn't change anything yet concerning VS/AP.
We can do the reset etc in JS. No need for an external app.
Will discuss what we can do.

**Comment by dga711** [comment]
Both issues should be resolve in 0.8.5
